torch/_dynamo/eval_frame.py

```diff
@@ -516,13 +626,17 @@ def catch_errors_wrapper(callback, hooks: Hooks):
     def catch_errors(frame, cache_entry, frame_state):
         assert frame_state is not None
 
+        is_skipfile = skipfiles.check(frame.f_code.co_filename)
         if (
             # TODO: the first condition is not covered by any test
             frame.f_lasti >= first_real_inst_idx(frame.f_code)
-            or skipfiles.check(frame.f_code.co_filename)
+            or is_skipfile
             or config.disable
         ):
-            log.debug("skipping %s %s", frame.f_code.co_name, frame.f_code.co_filename)
+            if not is_skipfile or config.verbose:  # don't log verbose skipfile skips
```

Comment:
This log was getting very verbose due to tracing many new function calls introduced by this PR. It is impossible to recursively `disable` these function calls due to recursivity (the new function calls are used in `_TorchDynamoContext` baseclass).

Reply:
This is fine for now